source/frontend/src/assets/scss/App.scss

```diff
@@ -10,13 +10,13 @@
 }
 
 .Toastify__toast.Toastify__toast--warning {
-  color: $text-color;
-  background-color: $light-accent-color;
+  color: var(--typography-color-secondary);
```

Comment:
could this be done using theme instead? Something like:
color: ${props => props.theme.bcTokens.typographyColorDanger};

Reply:
No. scss files don't have access to the theme. We would need to refactor all files using scss into styled components, which is outside the scope of this palette swap story (in my opinion).